services/QuillLMS/app/controllers/concerns/growth_results_summary.rb

```diff
@@ -0,0 +1,151 @@
+module GrowthResultsSummary
+  extend ActiveSupport::Concern
+
+  NOT_PRESENT = 'Not present'
+  NOT_CORRECT = 'Not correct'
+  FULLY_CORRECT = 'Fully correct'
+  PARTIALLY_CORRECT = 'Partially correct'
+
+  NO_PROFICIENCY = 'No proficiency'
+  PARTIAL_PROFICIENCY = 'Partial proficiency'
+  PROFICIENCY = 'Proficiency'
+  GAINED_PROFICIENCY = 'Gained proficiency'
+  MAINTAINED_PROFICIENCY = 'Maintained proficiency'
+
+  extend self
+
+  def growth_results_summary(current_user, pre_test_activity_id, post_test_activity_id, classroom_id)
```

Comment:
Do we have rspec unit tests for this method and the method in ResultsSummary?

Reply:
Not yet, see note in PR checklist!